int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -344,8 +356,23 @@ if (window.location.href.toLowerCase().match(/checkout/)) {
 if (edqEmailLineSelector) { edqEmailLineSelector.addEventListener("mouseover", function() {enableButtonDisable(edqCurrentSubmitButtonSelector, false);}); }
 if (edqPhoneLineSelectors) { edqPhoneLineSelectors.forEach(function(phoneSelector) { phoneSelector.addEventListener("mouseover", function() {enableButtonDisable(edqCurrentSubmitButtonSelector, false);}); }); }
 function edqEmailPhoneValidationCallback() {
-	if ((edqEmailEnable) && (edqEmailLineSelector)) { edqEmailValidationCallback(); }
-	if ((edqPhoneEnable) && (edqPhoneLineSelectors)) { edqPhoneValidationCallback(); }
+	/** Email Validation not restricting access when Phone Validation is activated in the Billing stage for SFRA
+	* Added the checkoutStage to review the button that we're going to disable to prevent on going to the next page on 
+	* the checkout page when the option is available in the Business Manager configuration.
+	* For more information see Bug #147798 */
+	var urlParams = new URLSearchParams(window.location.search);
+	var checkoutStage = urlParams.get('stage');
+	if (checkoutStage == "shipping")
+		setEdqButtonSelector("shipping");
```

Comment:
Since you have brackets on your other `if` statements, let's just make it consistent and always have brackets for all conditionals.